closed/src/java.base/share/native/libjncrypto/NativeCrypto.c

```diff
@@ -31,37 +31,28 @@
 #include <stdio.h>
 #include <stdlib.h>
 #include <string.h>
-#include <assert.h>
 
 #include "jdk_crypto_jniprovider_NativeCrypto.h"
 
 /* Structure for OpenSSL Digest context */
 typedef struct OpenSSLMDContext {
     EVP_MD_CTX *ctx;
     const EVP_MD *digestAlg;
-    unsigned char* nativeBuffer;
 } OpenSSLMDContext;
 
-/* Structure for OpenSSL Cipher context */
-typedef struct OpenSSLCipherContext {
-    EVP_CIPHER_CTX *ctx;
-    unsigned char* nativeBuffer;
-    unsigned char* nativeBuffer2;
-} OpenSSLCipherContext;
-
 /* Handle errors from OpenSSL calls */
-static void handleErrors(void) {
-    unsigned long errCode;
-
-    printf("An error occurred\n");
+static void printErrors(void) {
+    unsigned long errCode = -1;
```

Comment:
A negative number is not a reasonable initial value for an unsigned variable.

Reply:
changed to 0